include/procfile.bash

```diff
@@ -18,7 +18,7 @@ procfile-exec() {
 	procfile-load-env
 	procfile-load-profile
 	cd "$app_path"
-	unprivileged /bin/bash -c "$(eval echo $@)"
+	exec setuidgid "$unprivileged_user" $(eval echo $@)
```

Comment:
using `exec unprivileged $(eval echo $@)` in `procfile.bash` yields the following. 
````
/tmp/bashenv.304636478: line 410: exec: unprivileged: not found
```

changing `unprivileged()` in `herokuish.bash` to `exec setuidgid "$unprivileged_user" "$@"` yields an empty `/app/.profile.d/` and then of course this error when attempting to start.
```
setuidgid: fatal: unable to run node: file does not exist
```

digging into this scenario more we seem to never execute anything past this call in buildpack.bash
```
unprivileged "$selected_path/bin/compile" "$build_path" "$cache_path" "$env_path"
```

I added debugging near line 106 (the call to compile). 
```
	echo "[BUILDPACK.bash] buildpack-execute pre compile"
	unprivileged "$selected_path/bin/compile" "$build_path" "$cache_path" "$env_path"
	echo "[BUILDPACK.bash] buildpack-execute post compile"
```

Result when `unprivileged()` is `exec setuidgid "$unprivileged_user" "$@"`:
```
-----> Node.js app detected
       [BUILDPACK.bash] buildpack-execute pre compile

-----> Creating runtime environment
...
-----> Build succeeded!
       └── express@4.13.3

-----> Discovering process types
       Procfile declares types -> web
```

Result when `unprivileged()` is `setuidgid "$unprivileged_user" "$@"`:
```
-----> Node.js app detected
       [BUILDPACK.bash] buildpack-execute pre compile

-----> Creating runtime environment
...
-----> Build succeeded!
       └── express@4.13.3

       [BUILDPACK.bash] buildpack-execute post compile
-----> Discovering process types
       Procfile declares types -> web
```

any thoughts on why this might happen? i'm kinda stumped myself....